pysmt/smtlib/script.py

```diff
@@ -176,6 +177,20 @@ def serialize_to_string(self, daggify=True):
         self.serialize(buf, daggify=daggify)
         return buf.getvalue()
 
+class GoalsManager(object):
+    def __init__(self):
+        self.goals = []
+
+    def add(self, g):
+        self.goals.append(g)
+
+    def p(self):
```

Comment:
Cant' understand the meaning of this method:

- `rt` is useless
- it always returns None
- it is not documented